xcell/mappers/mapper_2MPZ.py

```diff
@@ -7,6 +7,8 @@
 
 
 class Mapper2MPZ(MapperBase):
+    map_name = '2MPZ'
```

Comment:
Why do we need this?

Reply:
To homogenize the  way we save the maps. See the MapperBase class